vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -992,6 +992,24 @@ import { ItemCache } from '@vaadin/vaadin-grid/src/vaadin-grid-data-provider-mix
         }));
       }));
 
+      grid.addEventListener('cell-focus', tryCatchWrapper(e => {
+        const eventContext = grid.getEventContext(e);
+
+        grid.dispatchEvent(new CustomEvent('grid-cell-focus', {
```

Comment:
Seems weird to add a generic prefix to indicate that this is the Flow specific event, but I'm not experienced enough to know about the existing naming conventions. Maybe someone more experienced like @tulioag can take a look?

Reply:
The pattern seems to be `affectedelement-eventname`. Ex: `item-click`,  `column-drag-resize`

Reply:
OK, seems to be correct then.